content/reference-transaction-format.md

```diff
@@ -724,6 +725,125 @@ In the above example with a ¥95/$15 offer and a ¥5/$2 offer, the situation is
 
 
 
+## PaymentChannelClaim
+[[Source]<br>](https://github.com/ripple/rippled/blob/develop/src/ripple/app/tx/impl/PayChan.cpp "Source")
+
+_Requires the [PayChan Amendment](concept-amendments.html#paychan)._
+
+Claim XRP from a payment channel, adjust the payment channel's expiration, or both. This transaction can be used differently depending on the transaction sender's role in the specified channel:
+
+The **source address** of a channel can:
+
+- Send XRP from the channel to the destination with _or without_ a signed Claim.
+- Set the channel to expire as soon as the channel's `SettleDelay` has passed.
+- Clear a pending `Expiration` time.
+- Close a channel immediately, with or without processing a claim first. The source address cannot close the channel immediately if the channel has XRP remaining.
+
+The **destination address** of a channel can:
+
+- Receive XRP from the channel using a signed Claim.
+- Close the channel immediately after processing a Claim, refunding any unclaimed XRP to the channel's source.
+
+**Any address** sending this transaction can:
+
+- Cause a channel to be closed if its `Expiration` or `CancelAfter` time is older than the previous ledger's close time. Any validly-formed PaymentChannelClaim transaction has this effect regardless of the contents of the transaction.
+
+Example PaymentChannelClaim:
+
+```json
+{
+  "Channel": "C1AE6DDDEEC05CF2978C0BAD6FE302948E9533691DC749DCDD3B9E5992CA6198",
+  "Balance": "1000000",
+  "Amount": "1000000",
+  "Signature": "30440220718D264EF05CAED7C781FF6DE298DCAC68D002562C9BF3A07C1E721B420C0DAB02203A5A4779EF4D2CCC7BC3EF886676D803A9981B928D3B8ACA483B80ECA3CD7B9B",
+  "PublicKey": "32D2471DB72B27E3310F355BB33E339BF26F8392D5A93D3BC0FC3B566612DA0F0A"
+}
+```
+
+<!--{# TODO: replace the above example with one where the channel, pubkey, signature, and balance match #}-->
+
+| Field       | JSON Type | [Internal Type][] | Description                    |
+|:------------|:----------|:------------------|:-------------------------------|
+| `Channel`   | String    | Hash256           | The unique ID of the channel, as a 64-character hexadecimal string. |
+| `Balance`   | String    | Amount            | _(Optional)_ Total amount of XRP, in drops, delivered by this channel after processing this claim. Required to deliver XRP. Must be more than the total amount delivered by the channel so far, but not greater than the `Amount` of the signed claim. Must be provided except when closing the channel. |
+| `Amount`    | String    | Amount            | _(Optional)_ The amount of XRP, in drops, authorized by the `Signature`. This must match the amount in the signed message.  |
```

Comment:
I suspect `Amount` will be a source of confusion. `Amount` is the value that a channel's balance will be allowed to be adjusted to. For example, to authorize an initial 10 drops, the amount will be 10. To authorize an additional 10 drops, the amount will be 20.